### PR TITLE
[docs] Updates field listing to include new fields for workers

### PR DESCRIPTION
**Pull request by jacobbednarz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -151,6 +151,8 @@ $ logshare-cli --api-key=<snip> --api-email=<snip> --zone-name=example.com --lis
   "EdgePathingOp": "Indicates what type of response was issued for this request (unknown = no specific action)",
   "EdgePathingSrc": "Details how the request was classified based on security checks (unknown = no specific classification)",
   "EdgePathingStatus": "Indicates what data was used to determine the handling of this request (unknown = no data)",
+  "EdgeRateLimitAction": "string: The action taken by the blocking rule; empty if no action taken. (beta)",
```
Comment (stephane-cloudflare) [inline README.md]: This field is deprecated now

File: README.md [hunk 2]
```diff
@@ -151,6 +151,8 @@ $ logshare-cli --api-key=<snip> --api-email=<snip> --zone-name=example.com --lis
...
+  "EdgeRateLimitID": "uint64: The internal rule ID of the rate-limiting rule that triggered a block (ban) or simulate action. 0 if no action taken. (beta)",
```
Comment (stephane-cloudflare) [inline README.md]: This field is deprecated now

**Comment by jacobbednarz** [comment]
Let me re-run this and get all the new fields as there are a bunch of others missing too.

File: README.md (hunk 1 above)
Comment (jacobbednarz) [reply README.md]: I've removed this from the output despite it still appearing in the tool output which is fetched from the ELS endpoint.

File: README.md (hunk 2 above)
Comment (jacobbednarz) [reply README.md]: also removed

**Comment by jacobbednarz** [comment]
Should be up to date again @stephane-cloudflare

**Pull request merged**